decred/decrediton
# Add husky pre-commit hook to run yarn prettify & lint

This installs husky v5 https://typicode.github.io/husky/#/ 

It also adds `postinstall` script to`package.json` to run `husky install` after yarn install automatically as mentioned in the 
installation guide linked above.

To test I think you need you need to:
 - `yarn install`.
 - it should run `husky install` automatically. 
 - add some empty lines to any `.jsx`, `.js`, `.css` or `.json` file.
 - try to commit and the empty lines to should be prettified automatically.
